docs/extend/extend-apiml/onboard-static-definition.md

```diff
@@ -77,14 +79,13 @@ Define your service and API in YAML format in the same way as presented in the f
 
 **Example:**
 
-To define your service in YAML format, provide the following definition in a YAML file as in the following sample `petstore` service:
+To define your service in YAML format, provide the following definition in a YAML file as in the following sample `petstore` service.
+This configuration is minimal configuration necessary for the gateway to properly route the requests to the application. The application won't be visible in the Catalog using this configuration. More detailed configuration is in [Customize configuration parameters](onboard-static-definition.md#Customize configuration parameters):
 
 ```yaml
 services:
     - serviceId: petstore
       catalogUiTileId: static
-      title: Petstore Sample Service
-      description: This is a sample server Petstore service
```

Comment:
Why remove this data?

Reply:
To show really the minimal necessary configuration